EIPS/eip-4361.md

```diff
@@ -1,10 +1,10 @@
 ---
 eip: 4361
-title: Sign-In with Ethereum.
+title: Sign-In with Ethereum
 description: Standardizes off-chain authentication for Ethereum accounts to establish sessions.
```

Comment:
We try to discourage "standard" in titles and descriptions (since nearly every EIP is a standard for something or other):

```suggestion
description: Off-chain authentication for Ethereum accounts to establish sessions.
```